src/text/raw_text_reader.rs

```diff
@@ -1044,4 +1057,58 @@ mod reader_tests {
 
         Ok(())
     }
+
+    #[test]
```

Comment:
PR tour: these unit tests overlap heavily with the `ion-tests` integration but do not use the `Element` (DOM) API. They were helpful in exploring some errors surfaced by `element_test_vectors.rs`.